protected/humhub/modules/user/authclient/Twitter.php

```diff
@@ -20,49 +21,32 @@ protected function defaultViewOptions()
         return [
             'popupWidth' => 860,
             'popupHeight' => 480,
-            'cssIcon' => 'fa fa-twitter',
+            'cssIcon' => 'fab fa-twitter',
```

Comment:
fab ?

Reply:
`fab` is used in Font Awesome 5 `fa` is deprecated.

Reply:
Ok

Reply:
But https://github.com/humhub/humhub/blob/958d940da60080d1a2cf2f02b429826c865b5a79/composer.json#L40

version for HumHub is `>=4.7.0 <5.0.0`

Reply:
Sadly using `fa fa-twitter` or even `fa-twitter` wouldn't work anyways (Which are 4+), would also seem that they still have this issue within 5...

Reply:
@Felli Maybe we can change to FA 5.0 in the v1.3?